tcf_website/templates/course/course.html

```diff
@@ -34,6 +34,19 @@ <h2>{{ course.title }}</h2>
             <br/>
             <br/>
 
+            {% if not taught_this_semester %}
```

Comment:
Can't we use this instead?
https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#for-empty

Reply:
I don't think so, since these queries include previous semesters (we just hide them if they're in the current semester tab)

Reply:
Ahh you're right.